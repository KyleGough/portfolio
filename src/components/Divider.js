import React from 'react';

export default function Divider() {
    return (
        <hr className='m-8 h-px bg-divider' />
    );
}
