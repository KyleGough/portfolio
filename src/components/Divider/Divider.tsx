import React from 'react';

export const Divider: React.FC = () => <hr className="m-8 h-px bg-divider" />;
