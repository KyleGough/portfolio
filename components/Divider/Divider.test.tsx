import { render } from '@testing-library/react';
import React from 'react';

import { Divider } from './Divider';

describe('Divider component', () => {
  it('renders', () => {
    const { container } = render(<Divider />);
    expect(container).toMatchSnapshot();
  });
});
