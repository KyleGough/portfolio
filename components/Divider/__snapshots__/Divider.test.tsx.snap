// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider component renders 1`] = `
<div>
  <hr
    class="m-8 h-px bg-divider"
  />
</div>
`;
